ngraph/frontend/tensorflow/src/decoder_proto.cpp

```diff
@@ -27,59 +29,69 @@ const std::map<::tensorflow::DataType, ov::element::Type>& TYPE_MAP() {
 }
 }  // namespace
 
-std::shared_ptr<ov::Variant> DecoderTFProto::get_attribute(const std::string& name,
-                                                           const VariantTypeInfo& type_info) const {
+template <class T>
+bool is_type(const VariantTypeInfo& type_info) {
+    return type_info == VariantWrapper<T>::get_type_info_static();
+}
+
+template <class T>
+shared_ptr<VariantWrapper<T>> create_variant(const T& data) {
+    return make_shared<VariantWrapper<T>>(data);
+}
+
+shared_ptr<Variant> DecoderTFProto::get_attribute(const string& name, const VariantTypeInfo& type_info) const {
     auto attrs = decode_attribute_helper(name);
     if (attrs.empty()) {
         return nullptr;
     }
 
-    if (type_info == VariantWrapper<std::string>::get_type_info_static()) {
-        return std::make_shared<VariantWrapper<std::string>>(attrs[0].s());
-    } else if (type_info == VariantWrapper<int64_t>::get_type_info_static()) {
-        return std::make_shared<VariantWrapper<int64_t>>(attrs[0].i());
-    } else if (type_info == VariantWrapper<std::vector<int64_t>>::get_type_info_static()) {
-        std::vector<int64_t> longs;
+    if (is_type<string>(type_info)) {
+        return create_variant<string>(attrs[0].s());
+    }
+    if (is_type<int64_t>(type_info)) {
```

Comment:
`if` -> `else if`

Reply:
fixed in https://github.com/openvinotoolkit/openvino/pull/8477